kbl2vec/src/main/java/com/foursoft/harness/kbl2vec/transform/components/common/GeneralTechnicalPartSpecificationTransformer.java

```diff
@@ -47,12 +44,20 @@ public TransformationResult<VecGeneralTechnicalPartSpecification> transform(fina
                                                                                 final KblPart source) {
         final VecGeneralTechnicalPartSpecification specification = new VecGeneralTechnicalPartSpecification();
 
-        if (source instanceof final KblConnectorHousing connectorHousing) {
-            final StringToColorConverter colorConverter =
-                    context.getConverterRegistry().getStringToColorConverter();
+        final StringToColorConverter colorConverter =
+                context.getConverterRegistry().getStringToColorConverter();
 
+        if (source instanceof final KblConnectorHousing connectorHousing) {
             colorConverter.convert(connectorHousing.getHousingColour()).ifPresent(
                     specification.getColorInformations()::add);
+        } else if (source instanceof final KblCavitySeal kblCavitySeal) {
+            colorConverter.convert(kblCavitySeal.getColour()).ifPresent(
```

Comment:
```suggestion
            colorConverter.convert(kblCavitySeal.getColour())
            .ifPresent(specification.getColorInformations()::add);
```

I know these might be picky. But it is much easier to read, of stream/optional methods always start on a new line (besides the .stream() method itself).

Reply:
I agree. I formatted the file accordingly.